### PR TITLE
jerry-ennett adds wisconsin-rapids file

### LINKED ISSUE #16: jerry-ennett practices using GitHub Flow
Create a branch off master
Add commits
Open a pull request
Collaborate
Merge with at least one thumbs up.

### PR DESCRIPTION
**Pull request by jerry-ennett**
The purpose of this pull request is to demonstrate how to use git and GitHub.

This PR closes #16

### CODE REVIEW

File: jerry-ennett-wisconsin-rapids.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+# Wisconsin Rapids
+
+### Directions
+
+### Places of Interest
+
+### Businesses
+
+### Favorite team - the Pack
```
Comment (ryan-koshiol) [inline jerry-ennett-wisconsin-rapids.md:10]: Are you sure you don't want a different team?

Comment (ryan-koshiol) [reply jerry-ennett-wisconsin-rapids.md:10]: Do you have directions for the Directions section?

Comment (jerry-ennett) [reply jerry-ennett-wisconsin-rapids.md:10]: It is America's team

**Review by ryan-koshiol — APPROVED** [review]
Not my team and I'm an American!

**Pull request merged**